autorest/codegen/serializers/__init__.py

```diff
@@ -22,6 +31,12 @@
 from .metadata_serializer import MetadataSerializer
 from .request_builders_serializer import RequestBuildersSerializer
 from .patch_serializer import PatchSerializer
+from .sample_serializer import SampleSerializer
+from ..models import BodyParameter
+from .utils import SAMPLE_AAD_ANNOTATION, SAMPLE_KEY_ANNOTATION
+
+_LOGGER = logging.getLogger(__name__)
+_BLACK_MODE = black.Mode()
```

Comment:
if we want black to black the samples repo, we should add that to the black plugin

Reply:
make sense! Since it may conflict with https://github.com/Azure/autorest.python/pull/1291 so will update it later